content/guides/hosting/monitoring-usage/audit-logging.md

```diff
@@ -61,6 +61,8 @@ An instance admin can fetch the audit logs for your W&B instance using the follo
     - `anonymize`: if set to `true`, remove any PII; defaults to `false`
 2. Execute HTTP GET request on the constructed full API endpoint, either by directly running it within a modern browser, or by using a tool like [Postman](https://www.postman.com/downloads/), [HTTPie](https://httpie.io/), cURL command or more.
 
+The audit logs API is accessible by instance or organization admins using basic authentication with their API key. With basic authentication, send the HTTP request with the `Authorization` header that contains the word `Basic` followed by a space and a base64-encoded string for `username:password` where `password` is your API key. For example, to authorize as `demo:p@55w0rd`, the header should be `Authorization: Basic ZGVtbzpwQDU1dzByZA==`.
```

Comment:
```suggestion
An organization or instance admin can use basic authentication with their API key to access the audit logs API. Set the HTTP request's `Authorization` header to the string `Basic` followed by a space, then the base-64 encoded string in the format `username:API-KEY`. In other words, replace the username and API key with your values separated with a `:` character, then base-64-encode the result. For example, to authorize as `demo:p@55w0rd`, the header should be `Authorization: Basic ZGVtbzpwQDU1dzByZA==`.
```
What do you think about this edit? If you like it, maybe replace the SCIM paragraph at the same time?